px-kpi-list.html

```diff
@@ -138,7 +140,19 @@
        */
       footer: {
         type: String
-      }
+      },
+	 /**
+      * This will check if label or statusIcon or statusLabel is present or not
+	  * if not then don't show header else show header 
+      *
+      * @property size
```

Comment:
This is not property `size`.

Also please move this function out of the properties object.